shopgate/pwa
# Improved interaction with cart items and the coupon field

# Description
This ticket contains some improvements for the cart page. It brings a better user experience while interacting with the quantity pickers of cart items and the coupon field. The keyboard now always hide after submitting the coupon form. Additionally the toggling of the PaymentBar was improved to avoid flickering when the keyboard toggles.

Additionally the tap area around the ContextMenu was enlarged to prevent unintentional opening of products.

## Type of change


## How to test it

Add some products to the cart and play around with the quantity inputs and the coupon input. The keyboard should always reliably show and hide. It should hide after a coupon code was entered and before the feedback popup appears.
Also interact with the context menu of the cart items. It should now be easier to open it without unintentional opening of products.
